src/app/hooks/useClickTracker/helpers.js

```diff
@@ -0,0 +1,42 @@
+const isLeftClick = button => button === 0;
```

Comment:
Has this been lifted directly from the Web Core implementation? I wonder if we should offer to maintain this as a utility library in psammead for them to use rather than duplicating the code? Just a thought, I wouldn't block this PR.

Reply:
Yeah it has, I added a few methods and changed some naming though. I'll ask Inti 👍

Reply:
That sounds interesting -- would be happy with a utility library that gives us less to support 😁 It'd be interesting to see if your browser support list means you'll need to tweak this further (and if so, how much) compared to WebCore's [browser support list](https://github.com/bbc/web/blob/main/docs/testing/browser-device-test-list.md#desktop-browsers).

Reply:
@intih Awesome, I can make this into a utility library in psammead at some point, perhaps at the same time I investigate if any changes need to be made to cater for all the other browsers 👍